### PR TITLE
some ie10 compat fixes

### PR DESCRIPTION
**Pull request by collin**
Fixes flexbox/babel compilation in IE10.

### CODE REVIEW

**Comment by collin** [comment]
@spra85 @kand

**Comment by kand** [comment]
What does `transform-es2015-classes` do?

**Comment by collin** [comment]
It compiles ES2015 classes to ES5.

This configuration compiles them in a way that is friendlier to older runtimes.

File: .babelrc [hunk 1]
```diff
@@ -1,7 +1,9 @@
 {
   "presets": ["react", "es2015"],
   "plugins": [
-    "transform-object-assign"
+    "transform-object-assign",
```
Comment (collin) [inline .babelrc]: This transforms `Object.assign` to plain ES5.

File: .babelrc [hunk 2]
```diff
@@ -1,7 +1,9 @@
...
+    "transform-proto-to-assign",
```
Comment (collin) [inline .babelrc]: And this one transforms setting `__proto__` to regular property assignment. Older runtimes are not aware of `__proto__`.

**Comment by spra85** [comment]
:+1:

**Comment by kand** [comment]
:+1:

**Pull request merged**